infra-templates/k8s/30/rancher-compose.yml

```diff
@@ -1,6 +1,6 @@
 .catalog:
   name: Kubernetes
-  version: v1.7.0-rancher3
+  version: v1.7.0-rancher4
   description: Rancher Kubernetes service
   minimum_rancher_version: v1.6.3-rc1
```

Comment:
we need to update this to v1.6.5-rc1 as well as add in `upgrade_from` to prevent users from upgrading.